SDDC_FX3/makefile

```diff
@@ -107,4 +108,7 @@ $(ELF2IMG) : $(FX3FWROOT)/util/elf2img/elf2img.c
 $(MODULE).img: $(EXES) $(ELF2IMG)
 	$(ELF2IMG) -i $< -o $@ -v
 
+$(MODULE).h: $(EXES) $(ELF2IMG)
```

Comment:
what's this? why a image file use .h extension? hex?

Reply:
I've patched elf2img.c to generate .h file to include the firmware into the library as a BLOB. As we do not have persistent firmware storage (SPI flash), then it may be good to ship the firmware this way.
Suggest a better way of including the firmware into the library. Assembler .S file and `incbin` directive? Linker script?